spectrum-offchain/src/backlog.rs

```diff
@@ -231,13 +231,20 @@ where
 impl<TOrd, TStore> BacklogService<TOrd, TStore>
 where
     TOrd: OnChainOrder + Weighted + Hash + Eq,
+    TOrd::TOrderId: Debug,
     TStore: BacklogStore<TOrd>,
 {
-    pub fn new<TOrd0: IsEqual<TOrd>>(store: TStore, conf: BacklogConfig) -> Self {
+    pub async fn new<TOrd0: IsEqual<TOrd>>(store: TStore, conf: BacklogConfig) -> Self {
+        let mut pending_pq = PriorityQueue::new();
+        for ord in store.find_orders(|_| true).await {
```

Comment:
Why not just creat something like `get_all()` or `stream_all()`?

Reply:
Actually, it's currently used now to remove unnecessary orders once it's confirmed that no stakers are left for a given epoch: https://github.com/spectrum-finance/spectrum-offchain-ergo/blob/73ac89189fd755f91c24445b816c2f42c57c01c5/spectrum-offchain-lm/src/scheduler/process.rs#L68-L79